docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Caching in Next.js
+nav_title: Caching
+description: An overview of caching mechanisms in Next.js.
+---
+
+Next.js has a few caching mechanisms to improve your application's performance. This section provides an in-depth look at the different mechanisms, how they affect each other, and how you can use them.
+
+Please note that some caching heurestics are automatic, while others can be manually configured. Caching is also split between rendering cache, router cache, and data cache, client and server environments.
+
+## Overview
+
+{/* TO DO: Diagram showing how caching mechanisms come into play in the request lifecycle, going from  initial request, to server, to response, to router cache, to subsequent navigation with new request. */}
+
+| Mechanism             | What is cached?                   | Where is it cached? | Why is it cached?                               | Duration                                                                                                           | APIs                                                                         |
+| --------------------- | --------------------------------- | ------------------- | ----------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------- |
+| Rendering Cache       | React Components                  | Server (CDN)        | Reduce the rendering cost.                      | Persistent or revalidate lifetime                                                                                  | Route Segment Config, and Dynamic Functions                                  |
+| Router Cache          | Rendered Server Component Payload | Client (in-memory)  | Improve navigation performance.                 | Single Session (cleared by hard navigation or refresh), or 30s for dynamic routes, and 5 minutes for static routes | <Link>, router.prefetch, router.refresh, revalidatePath, revalidateTag       |
+| Data Cache            | Result of data requests           | Server (CDN)        | Store data across user requests and deployments | Persistent or revalidate lifetime                                                                                  | fetch.cache, fetch.revalidate, unstable_cache, revalidatePath, revalidateTag |
```

Comment:
I recognize that there is not only `CDN` but also `in-memory` depending on the conditions. I couldn't found a way to actually behave this, but [in this lines](https://github.com/vercel/next.js/blob/7ce663ed52a881ffb3fa38e50ff5d6e135293fbf/packages/next/src/server/lib/incremental-cache/file-system-cache.ts#L40-L59) and [this lines](https://github.com/vercel/next.js/blob/7ce663ed52a881ffb3fa38e50ff5d6e135293fbf/packages/next/src/server/lib/incremental-cache/fetch-cache.ts#L75-L94) here shows it.I apologize if I'm wrong.

```suggestion
| Data Cache            | Result of data requests           | Server (CDN or in-memory)        | Store data across user requests and deployments | Persistent or revalidate lifetime                                                                                  | fetch.cache, fetch.revalidate, unstable_cache, revalidatePath, revalidateTag |
```

Reply:
I think the default rendering cache and data cache store data in the file system. In-memory cache is mainly used to optimize the file system cache, and the typical application developer would not use it directly. And CDNs are used in special environments such as Vercel.
Therefore, I think it would be better to write "Server (file system)" there.

Reply:
@delbaoliveira I think these comments have already resolved as they were updated to just "Servers" (i.e.  omit the details).